### PR TITLE
hw

### PR DESCRIPTION
**Pull request by YaTut1901**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -9,6 +15,25 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        final char W = 'w';
+        ArrayList<String> wordsWithWList = new ArrayList<String>();
+        String text = "";
+        try {
+            text = new String(Files.readAllBytes((Paths.get(fileName)))).toLowerCase().replaceAll(
+                    "[^A-Za-zА-Яа-я ]", "");
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:23]: your solution has one drawback:
the last word on the line is sticked together with the first word on the next line

`word1 word2
word 3` 
=> `word1 word2word3`

let's try retrieving lines first, and then split each line

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -9,6 +15,25 @@
...
+        } catch (IOException e) {
+            System.out.println("Такого файла не существует");
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:25]: we should rethrow an informative RuntimeException here

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -15,21 +15,23 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        final char W = 'w';
+        final char CONSTW = 'w';
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:18]: add public and static keywords

File: src/main/java/core/basesyntax/FileWork.java [hunk 4]
```diff
@@ -14,8 +14,10 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    public static final char CONSTW = 'w';
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:18]:
```suggestion
    public static final char CONSTANT_W = 'w';
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 5]
```diff
@@ -8,7 +14,30 @@
...
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> wordsWithWList = new ArrayList<String>();
+        String text = "";
+        try {
+            text = new String(Files.readAllBytes((Paths.get(fileName)))).toLowerCase().replaceAll(
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:24]: let's use readAllLines() method. This way you will get List<String> and won't have to use split("\n") to divide text to lines

File: src/main/java/core/basesyntax/FileWork.java [hunk 6]
```diff
@@ -8,7 +14,30 @@
...
+                    "[^A-Za-zА-Яа-я\n ]", "");
+        } catch (IOException e) {
+            throw new RuntimeException("Такого файла не существует", e);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FileWork.java:27]: let's use english language in our programs)Anybody in the world should be able to understand that

**Pull request closed without merging**